app.js

```diff
@@ -5,6 +5,7 @@ var cookieParser = require("cookie-parser");
 var logger = require("morgan");
```

Comment:
Replace all `vars` with `const`. 

You should use `const` instead of `var` for importing modules in modern JavaScript. This is because `const` indicates that the variable will not be reassigned, which is appropriate for module imports. Using `var` can lead to issues with variable scope.

Reply:
Updated.